src/components/crossForm/CrossForm.vue

```diff
@@ -184,12 +184,20 @@
           </div>
         </div>
       </div>
+      <div class="row">
+        <div class="offset-md-2 col-md-8 col-12">
+          <p v-if="destinationNetwork.isRsk" class="alert alert-warning" role="alert">
+            Binance is not taking deposits sent by a smart contract for RSK network, they only
+            accept deposits from an account
+          </p>
+        </div>
+      </div>
     </div>
     <div v-if="claimCost" class="row justify-content-center mb-4">
       <div class="col-sm-12 col-md-5 border note-container px-4 py-2 bg-light text-center">
         <strong class="fw-bold d-block">Important!</strong>
         <span>
-          You'll need <strong>{{ claimCost }} ETH</strong> to claim the tokens
+          You'll need <strong>{{ claimCost }}</strong> to claim the tokens
```

Comment:
shouldn't we be able to show ETH/BNB depending on the chain?

Reply:
because we're not being clear to the user I think

Reply:
Done at https://github.com/rsksmart/tokenbridge-ui/commit/2945d716db8fe08c8b0e21aa2a9d3af81afe7978